checks.d/postgres.py

```diff
@@ -285,9 +302,9 @@ def _collect_stats(self, key, db, instance_tags, relations, custom_metrics):
                 self.SIZE_METRICS
             ]
 
-        # Only available for >= 9.1 due to
-        # pg_last_xact_replay_timestamp
-        if self._is_9_1_or_above(key,db):
+        # Only available for >= 9.2 due to
+        # pg_xlog_location_diff
+        if self._is_9_2_or_above(key,db):
```

Comment:
Looks like you will break replication metrics for PG 9.1 users.

You should append the replication_delay_bytes metrics where PG >= 9.2 and keep replication_delay in case of 9.1